main.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+from database import SessionLocal
+import models
+from schemas import AuthorCreate, BookCreate
+import crud
+
+app = FastAPI()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+@app.post("/authors/", response_model=AuthorCreate)
+def create_author(author: AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
+
+@app.get("/authors/", response_model=List[AuthorCreate])

```

Comment:
The response model for the `get_authors` endpoint should be `List[Author]` instead of `List[AuthorCreate]` to include the `id` and `books` fields in the response.